alf/examples/mbrl_reward_functions.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2020 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gin
+import torch
+
+from alf.utils import common
+# implement the respective reward functions for desired environments here
+
+
+@gin.configurable
+def reward_function_for_pendulum(obs, action):
```

Comment:
There is already a reward function for pendulum? It seems you are trying to organizing all mbrl reward functions in a single file here.